test/test_other.py

```diff
@@ -133,12 +138,17 @@ def also_with_wasmfs_all_backends(f):
   def metafunc(self, backend, *args, **kwargs):
     if backend:
       self.setup_wasmfs_test()
+      if backend == 'node':
+        self.setup_nodefs_test()
+      elif backend == 'raw':
+        self.setup_noderawfs_test()
       self.emcc_args.append(f'-D{backend}')
     f(self, *args, **kwargs)
 
   parameterize(metafunc, {'': (None,),
-                          'wasmfs': ('WASMFS_MEMORY_BACKEND',),
-                          'wasmfs_node': ('WASMFS_NODE_BACKEND',)})
+                          'wasmfs': ('memory',),
+                          'wasmfs_node': ('node',),
+                          'wasmfs_raw': ('raw',)})
```

Comment:
@tlively do you think it makes sense to test both nodefs (where nodefs is mounted as `/nodefs`) and noderawfs (where nodefs is mounted at the root) with wasmfs?  I guess maybe yes?

Reply:
No strong opinion, but I would err on the side of testing both, yeah.